tools/litmus_tests/generations/gen_oldest_arm64.litmus

```diff
@@ -0,0 +1,15 @@
+AArch64 wt_gen_oldest
+{
+conn_gen=5;sess_gen=0;
+0:X1=conn_gen; 0:X3=sess_gen;
+1:X1=conn_gen; 1:X3=sess_gen;
+2:X1=conn_gen; 2:X3=sess_gen;
+}
+
+ P0            | P1           | P2                   ;
+ LDR W0, [X1]  | LDR W0, [X1] | MOV W9, #1           ;
```

Comment:
We could simplify this using X registers everywhere and less random register numbers if you wish.

Reply:
I'll push up a simplification

Reply:
[5b86b30](https://github.com/wiredtiger/wiredtiger/pull/9463/commits/5b86b3009c1ebe9d6b8bcc22a5a477de4c1645e1)